lib/ansible/plugins/action/__init__.py

```diff
@@ -747,7 +749,7 @@ def _low_level_execute_command(self, cmd, sudoable=True, in_data=None, executabl
         out = self._strip_success_message(out)
 
         display.debug("_low_level_execute_command() done: rc=%d, stdout=%s, stderr=%s" % (rc, stdout, stderr))
-        return dict(rc=rc, stdout=out, stdout_lines=out.splitlines(), stderr=err)
+        return dict(rc=rc, stdout=out, stdout_lines=out.splitlines(), stderr=err, stderr_lines=err.splitlines())
```

Comment:
@nitzmahone Thanks for the comment. This is the first actual python I've written. I'll have to do some research and figure out what your comment means.